### ISSUE TITLE
Weitere Vokabulare via Wrapper einbinden

### ISSUE THREAD

**Issue by nichtich** [gbv/uri-gbv-terminology#12]
BARTOC, VIAF, GND, ORCID etc.

**Comment by StiftungAusNachlass** [comment]
Ja, unbedingt. Ich wollte deswegen schon zu dir runter kommen.. :-) Schön, dass es da ein Issue gibt, ich helfe auch gern und würde mit VIAF anfangen wollen.

Ist das hier https://jskos-php-examples.herokuapp.com/VIAF.php?uri=http%3A%2F%2Fviaf.org%2Fviaf%2F87772061 nur ein Beispiel, oder ein funktionierender Dienst, den wir einfach einbinden können? Habe da nur diesen Bookmark zu und weiß, dass das von dir kommt.. 👍
